tests/structured_conf/test_structured_config.py

```diff
@@ -800,7 +800,16 @@ def test_set_list_correct_type(self, module: Any) -> None:
         assert cfg.tuple == value
 
     @mark.parametrize(
-        "value", [1, True, "str", 3.1415, ["foo", True, 1.2], User(), [None]]
+        "value",
+        [
+            1,
+            True,
+            "str",
+            3.1415,
+            ["foo", True, 1.2],
+            User(),
+            # [None],  # expected failure, https://github.com/omry/omegaconf/issues/579
```

Comment:
You can indicate to [pytest that this is an expected failure](https://docs.pytest.org/en/stable/parametrize.html) (look for pytest.mark.xfail).

Reply:
Done in https://github.com/omry/omegaconf/pull/705/commits/c4dd99d390f1e7e56bb2e6561499a6958ad77107.